src/pages/News/index.js

```diff
@@ -1,9 +1,135 @@
 import React, { Component } from "react";
+import windowSize from "react-window-size";
+
+import articles from "../../fixtures/articles";
+import "./style.css";
+import ArrowLeft from "../../icons/ArrowLeft";
+import ArrowRight from "../../icons/ArrowRight";
+import NewsArticles from "../../components/NewsArticles";
 
 class News extends Component {
+  state = {
+    articlesList: articles,
+    newArticles: [],
+    articlesLength: 0,
+    count: 0,
+    sliceArticle: 0,
+		smallMonitor : 5,
+		bigMonitor : 11
+  };
+
+  componentDidMount() {
+    const { articlesList } = this.state;
+    let expansionMonitor = 0;
+    this.props.windowWidth <= 548
+      ? (expansionMonitor = 5)
+      : (expansionMonitor = 11);
+    let newArticles = articlesList.slice(0, expansionMonitor);
+    this.setState({
+      newArticles: newArticles,
+      articlesLength: articlesList.length,
+      sliceArticle: expansionMonitor
+    });
+  }
```

Comment:
this.setState({
      newArticles,
      articlesLength: articlesList.length,
      sliceArticle: expansionMonitor
    });